redisinsight/ui/src/pages/browser/components/bulk-actions/BulkActionSummary/BulkActionSummary.tsx

```diff
@@ -16,10 +16,10 @@ export interface Props {
   'data-testid': string
 }
 
-const SummaryContainer = styled(Row)`
+const SummaryContainer = styled(RiRow)`
   padding-top: 18px;
 `
```

Comment:
I'm just brainstorming here, but what if we introduce https://www.npmjs.com/package/styled-system ?

What is the problem that bugs me: We'll introduce (and we already did) a lot of components with small styling fixes, causing a lot of boilerplate:

```
const RiRow = ... // this is OK

// This is necessary because we can't use <RiRow pt="18px" />
const SummaryContainer = styled(RiRow)`
  padding-top: 18px;
`

// Our end goal would be to remove the hardcoded pixels
<RiRow pt="small" /> or <RiRow pt={3} />
```

So what do you think?

Ofc, this is dead project, so as styled-components, but I guess it will help us if we migrate to something similar to tailwindcss in the future.